.travis.yml

```diff
@@ -16,6 +16,7 @@ matrix:
 
 script:
   - cargo build --features="$FEATURES"
+  - cargo build
```

Comment:
Sorry, I think we've been talking past each other on this.  I meant you should add:

```
   - cargo build --features="$FEATURES" -p rayon-demo
```

... leaving the matching `cargo test` under nightly only.